platform/src/main/java/org/hillview/table/api/ITable.java

```diff
@@ -77,4 +78,48 @@
      * Formats the first rows in the table as a long string.
      */
     String toLongString(int rowsToDisplay);
+
+    /**
+     * Returns a DoubleMatrix that is a column vector corresponding to the given column name.
+     * This copies the entire column into the new matrix.
+     *
+     * @param colName Name of the column that is desired.
+     * @param converter String converter to use for string data.
+     * @return The matrix with the numeric data from the specified column.
+     */
+    default DoubleMatrix getNumericColumn(String colName, IStringConverter converter) {
```

Comment:
I'll remove it from ITable for now. Perhaps I'll add the functionality later (elsewhere) when we need JBLAS for the nontrivial linear algebra.